demo/eth_verifier/verify.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export CONTRACT_ADDR=0x0 # replace this after deploying
```

Comment:
we should receive this value as an argument in the invocation

Reply:
done!